### PR TITLE
about dialog: deduplicate code

### PR DESCRIPTION
**Pull request by sebastiankutschbach**

### CODE REVIEW

File: lib/ui/about.dart [hunk 1]
```diff
@@ -69,134 +70,61 @@ class AboutScreenState extends State<AboutScreen> {
...
-            Flexible(
-              child: Text(
-                _enforcedTimeZone,
-                style: _valueStyle,
-                maxLines: 10,
-                overflow: TextOverflow.ellipsis,
-                textAlign: TextAlign.center,
-              ),
-            ),
+            ..._valueWithTitle(title: 'Version:', value: _version),
```
Comment (MrCsabaToth) [inline lib/ui/about.dart:72]: Oh nice, so you unroll the list with the `...` operator!

**Review by MrCsabaToth — APPROVED** [review]
Cutting down plumbing code is always good.

**Comment by MrCsabaToth** [comment]
BTW I have a line length 100 because I found that 80 caused a little too many line breaks. But since 100 is not the default I see you ran into that. Should the default 80 line length be considered? I'm not hard hat on that 100.

**Review by MrCsabaToth — APPROVED** [review]
Yeah, I have that 100 line length formatting... sorry that you ran into that

**Comment by sKutschbachVW** [comment]
Regarding the line length: I dont have a strong opinion regarding that. I can understand your concern regarding the line breaks but on the other hand I see that using default settings would make contribution much easier.

**Comment by sebastiankutschbach** [comment]
Regarding the line length: I dont have a strong opinion regarding that. I
can understand your concern regarding the line breaks but on the other hand
I see that using default settings would make contribution much easier.

Am Do., 10. März 2022 um 18:20 Uhr schrieb Csaba Toth <
***@***.***>:

**Pull request merged**